miracum/MIRACUM_-_main_(beta3).ga

```diff
@@ -0,0 +1,2908 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "format-version": "0.1",
+    "name": "MIRACUM - main (beta3)",
```

Comment:
hu, is this the latest version @wm75 ?

Reply:
yes, that should be it. Should probably have removed the beta when I last updated it, but wasn't so sure it would be the released version :)